src/locales/en/loginForm.json

```diff
@@ -0,0 +1,12 @@
+{
+  "loginTitle": ["Dorm room social manager", "LOGIN"],
```

Comment:
let's create a `title` and `shortTitle` instead of array of tittles

Reply:
should be fixed in 9e244cb